time stamp

float32 steer_1_Volt
float32 roll_1_Volt
float32 steer_2_Volt
float32 roll_2_Volt
float32 steer_3_Volt
float32 roll_3_Volt
float32 steer_4_Volt
float32 roll_4_Volt
float32 steer_1_Amp
float32 roll_1_Amp
float32 steer_2_Amp
float32 roll_2_Amp
float32 steer_3_Amp
float32 roll_3_Amp
float32 steer_4_Amp
float32 roll_4_Amp

